src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -73,11 +85,15 @@ public function getBlockPrefix()
     /**
      * Returns a normalized array of timezone choices.
      */
-    private static function getTimezones(int $regions): array
+    private static function getTimezones(int $regions, string $input): array
     {
         $timezones = [];
 
         foreach (\DateTimeZone::listIdentifiers($regions) as $timezone) {
+            if ('intltimezone' === $input && 'Etc/Unknown' === \IntlTimeZone::createTimeZone($timezone)->getID()) {
+                continue;
```

Comment:
for reference, the following PHP timezone IDs become unavailable:

```
America/Punta_Arenas
Asia/Atyrau
Asia/Famagusta
Asia/Qostanay
Asia/Tomsk
Asia/Yangon
Europe/Kirov
Europe/Saratov
```

(see https://github.com/unicode-org/icu/blob/master/icu4c/source/data/misc/metaZones.txt for expiration dates)